### PR TITLE
Remove reload icon button from dashboard and reminder list pages

### PR DESCRIPTION
**Pull request by StanislavPrime**
Remove the reload icon button from dashboard and reminder list pages. Ticket : https://github.com/Prime-Holding/rx_bloc/issues/215

### CODE REVIEW

File: examples/reminders/lib/feature_dashboard/views/dashboard_page.dart [hunk 1]
```diff
@@ -81,14 +81,7 @@ class DashboardPage extends StatelessWidget implements AutoRouteWrapper {
...
 
-  AppBar _buildAppBar(BuildContext context) => AppBar(
-        actions: [
-          IconButton(
-            icon: const Icon(Icons.refresh),
-            onPressed: context.read<DashboardBlocType>().events.fetchData,
-          ),
-        ],
-      );
+  AppBar _buildAppBar(BuildContext context) => AppBar();
```
Comment (StanevPrime) [inline examples/reminders/lib/feature_dashboard/views/dashboard_page.dart:84]: Do we really need a method that returns a single widget ?

File: examples/reminders/lib/feature_reminder_list/views/reminder_list_page.dart [hunk 2]
```diff
@@ -39,17 +39,7 @@ class ReminderListPage extends StatelessWidget implements AutoRouteWrapper {
...
-          IconButton(
-            icon: const Icon(Icons.refresh),
-            onPressed: () => context
-                .read<ReminderListBlocType>()
-                .events
-                .loadPage(reset: true),
-          ),
-        ],
-      );
+  AppBar _buildAppBar(BuildContext context) => AppBar();
```
Comment (StanevPrime) [inline examples/reminders/lib/feature_reminder_list/views/reminder_list_page.dart:42]: Same comment as above

**Review by StanevPrime — CHANGES_REQUESTED** [review]

File: examples/reminders/lib/feature_dashboard/views/dashboard_page.dart (hunk 1 above)
Comment (StanislavPrime) [reply examples/reminders/lib/feature_dashboard/views/dashboard_page.dart]: I have removed the method and moved the `AppBar()` in the `Scaffold`

File: examples/reminders/lib/feature_reminder_list/views/reminder_list_page.dart (hunk 2 above)
Comment (StanislavPrime) [reply examples/reminders/lib/feature_reminder_list/views/reminder_list_page.dart]: I have removed the method and moved the `AppBar()` in the `Scaffold`

**Pull request merged**